File: ChatGPT/kotlin_to_java/architecture-samples_Diff/architecture-samples-515e82c9/prev/app/src/test/java/com/example/android/architecture/blueprints/todoapp/TestUtil.java
package com.example.android.architecture.blueprints.todoapp;

import androidx.lifecycle.LiveData;
import org.junit.Assert;

public class TestUtil {
    public static void assertLiveDataEventTriggered(
        LiveData<Event<String>> liveData,
        String taskId
    ) {
        Event<String> value = LiveDataTestUtil.getValue(liveData);
        Assert.assertEquals(value.getContentIfNotHandled(), taskId);
    }

    public static void assertSnackbarMessage(LiveData<Event<Integer>> snackbarLiveData, int messageId) {
        Event<Integer> value = LiveDataTestUtil.getValue(snackbarLiveData);
        Assert.assertEquals(value.getContentIfNotHandled(), messageId);
    }
}

File: ChatGPT/kotlin_to_java/architecture-samples_Diff/architecture-samples-515e82c9/curr/app/src/test/java/com/example/android/architecture/blueprints/todoapp/TestUtil.kt

package com.example.android.architecture.blueprints.todoapp;

import androidx.lifecycle.LiveData;
import org.junit.Assert;

public class Utils {

    public static void assertLiveDataEventTriggered(
            LiveData<Event<String>> liveData,
            String taskId
    ) {
        Event<String> value = liveData.awaitNextValue();
        Assert.assertEquals(value.getContentIfNotHandled(), taskId);
    }

    public static void assertSnackbarMessage(
            LiveData<Event<Integer>> snackbarLiveData,
            int messageId
    ) {
        Event<Integer> value = snackbarLiveData.awaitNextValue();
        Assert.assertEquals(value.getContentIfNotHandled(), messageId);
    }
}
